packages/blade/src/components/Button/Button/__tests__/__snapshots__/Button.web.test.tsx.snap

```diff
@@ -1,18 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`<Button /> should render button with default properties 1`] = `
-.c2 {
```

Comment:
Kinda wierd, the style snapshots just decided to reorder all of these .c2 classes and put it after .c1